src/tools/opt-dist/src/exec.rs

```diff
@@ -132,10 +132,13 @@ impl Bootstrap {
     }
 
     pub fn llvm_pgo_instrument(mut self, profile_dir: &Utf8Path) -> Self {
```

Comment:
Shouldn't you be modifying `llvm_pgo_optimize` instead? Also, please use `Option<LlvmPGOProfile>` instead of passing the profile an empty string.